api/src/main/java/io/cloudevents/common/BinaryData.java

```diff
@@ -0,0 +1,58 @@
+package io.cloudevents.common;
+
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
It would be nice to remove all external deps from API. This will avoid any kind of conflict with pre-existing libs or frameworks.

When someone wants to use JSON, they are concerned about Jackson usage. Checkout this issue #113.

Reply:
look this thread https://github.com/cloudevents/sdk-java/pull/108#discussion_r411153130